app/controllers/items_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ItemsController < ApplicationController
+  def index
+    if params[:merchant_id].nil?
+      @items = Item.all
+    else
+      @merchant = Merchant.find(params[:merchant_id])
+
+      @items = @merchant.items
+    end
+  end
+
+  def show
+    @item = Item.find(params[:id])
+    @merchant = Merchant.find(@item.merchant_id)
+  end
+
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create
+    binding.pry
+  end
```

Comment:
This would have been user story 10. Let's say your form had all the fields for an item. When you filled those fields in and clicked the button to submit, you'd be taken to the route that your field chose. Ideally, the route would be something like "/merchants/:merchant_id/items" because we want to create an item, but we also want that merchant id, so going to a route where we need to actually pass the merchant id in the path helps us in our create method becuase then we have an id to look up a merchant by and add that item to that merchant. So your create method would look something like this assuming you had a form that passed you all of the fields for the new item: 

```ruby 
def create 
merchant = Merchant.find(params[:merchant_id])
merchant.items.create(item_params)
redirect_to "/merchants/#{merchant.id}"
end 
```